### ISSUE TITLE
[TF2] detonated caber deals only 55 damage instead of stock 65 with no mention of it in the stats

### ISSUE THREAD

**Issue by pfroncz** [ValveSoftware/Source-1-Games#4918]
Just as the title says, the detonated caber hit deals onlt 55 damage and it's not listed in it's stats that it deals less damage, but idk if it's just the text missing or the caber Should deal 65 damage on detonated mele hit, either way,  it should be fixed

**Comment by JoriKos** [comment]
This is intentional, it's simply missing the 15% damage penalty attribute.

**Comment by pfroncz** [comment]
> 

Yeah i figured it out, but the attribute should be added to it's description in order to not confuse newer players
